scripts/install.sh

```diff
@@ -69,31 +77,107 @@ printf "Network: $NETWORK\n"
 
 cd $KEEP_BEACON_SOL_PATH
 
-printf "${LOG_START}Installing YARN dependencies...${LOG_END}"
-yarn install
+printf "${LOG_START}Installing beacon YARN dependencies...${LOG_END}"
+yarn
```

Comment:
I liked `yarn install` more as it was explicit.
```suggestion
yarn install
```

Reply:
ok